### PR TITLE
Arbitrum deployment

### PR DESCRIPTION
**Pull request by pettinarip**

### CODE REVIEW

**Comment by samajammin** [comment]
Hey @pettinarip is this ready for review? Seems like it's good to merge?

**Comment by daodesigner** [comment]
Add setting env file as required to finalize in docs

File: contracts/scripts/finalize.ts [hunk 1]
```diff
@@ -1,14 +1,26 @@
...
-  const [deployer] = await ethers.getSigners()
-  const state = JSON.parse(fs.readFileSync('state.json').toString())
+  let fundingRoundAddress, coordinator
+  if (network.name === 'localhost') {
+    const state = JSON.parse(fs.readFileSync('state.json').toString())
+    fundingRoundAddress = state.factory
+
+    const signers = await ethers.getSigners()
+    coordinator = signers[1]
+  } else {
```
Comment (daodesigner) [inline contracts/scripts/finalize.ts:13]: Explain this switch in the docs

**Comment by pettinarip** [comment]
> Hey @pettinarip is this ready for review? Seems like it's good to merge?

Ready for review, ty

File: contracts/.env.example [hunk 2]
```diff
@@ -8,12 +8,17 @@ BRIGHTID_CONTEXT=clr.fund
...
 WALLET_MNEMONIC=
 WALLET_PRIVATE_KEY=
 
 # Token address for funding round .setToken
 NATIVE_TOKEN_ADDRESS=
+
+# Required to use the tally and finalize scripts
+ROUND_ADDRESS=
+COORDINATOR_PK=
+COORDINATOR_ETH_PK=
```
Comment (daodesigner) [inline contracts/.env.example]: Note setting these env. variables in the finalize script docs

**Review by daodesigner — CHANGES_REQUESTED** [review]
nit: add new env variables used in finalize scripts to docs

File: contracts/.env.example (hunk 2 above)
Comment (pettinarip) [reply contracts/.env.example]: To be super clear, which docs exactly are you talking about? the README.md or to add comments on top of the function itself, in the `scripts/finalize.ts` file, or something else?

Comment (daodesigner) [reply contracts/.env.example]: Valid question, I meant in the coordinator docs: https://github.com/ethereum/clrfund/blob/develop/docs/coordinator.md

**Comment by daodesigner** [comment]
Lgtm

File: docs/coordinator.md [hunk 3]
```diff
@@ -93,6 +93,38 @@ Finally, the [CID](https://ipfs.io/ipns/docs.ipfs.io/concepts/content-addressing
 await fundingRound.publishTallyHash('<CID>')
 ```
 
+## Coordinate using Docker
+
+In case you are in a different OS than Linux, you can run all the previous MACI CLI commands by running the Docker image located in the MACI repo.
+
+**Note:** the [x32 params](https://gateway.pinata.cloud/ipfs/QmWSxPBNYDtsK23KwYdMtcDaJg3gWS3LBsqMnENrVG6nmc) have been tested using Ubuntu 21.04 + Node 15.8.0.
```
Comment (pettinarip) [inline docs/coordinator.md:100]: @daodesigner https://github.com/clrfund/monorepo/issues/417#issuecomment-1001791628

**Review by samajammin — APPROVED** [review]

**Pull request merged**